packages/blade/src/components/Typography/BaseText/BaseText.native.tsx

```diff
@@ -13,30 +13,29 @@ const StyledBaseText = styled.Text<StyledBaseTextProps>(
     fontWeight,
     fontStyle,
     textDecorationLine,
+    numberOfLines,
     lineHeight,
     textAlign,
     as,
     ...props
   }) => {
     const styledPropsCSSObject = useStyledProps(props);
-    if (as) {
-      throw new Error(`[Blade: BaseText]: "as" prop is not supported for BaseText on React Native`);
-    } else {
```

Comment:
Ignoring on RN because even if we pass as prop there is no breakage or side effects. This reduces a lot of checks in typography components because I don't need to manually do `isReactNative ? undefined : as` in places.